libtsuba/src/RDGTopology.cpp

```diff
@@ -146,11 +146,7 @@ RDGTopology::Map() {
   // EdgeTypeAwareTopologies have a larger adj_indices array than usual topologies
   if (topology_state_ ==
       tsuba::RDGTopology::TopologyKind::kEdgeTypeAwareTopology) {
-    KATANA_LOG_VASSERT(
-        edge_condensed_type_id_map_size_,
-        "mapping an EdgeTypeAwareTopology with a zero sized "
-        "condensed_type_id_map");
-    adj_indices_size = num_nodes_ * edge_condensed_type_id_map_size_;
+    adj_indices_size = std::max(num_nodes_, num_nodes_ * edge_condensed_type_id_map_size_);
```

Comment:
nit: This hides the reason for using max. FWIW I prefer to keep it explicit even if it is more verbose:
    `adj_indices_size = num_nodes_ == 0 ? 0 : num_nodes_ * edge_condensed_type_id_map_size_`;